### ISSUE TITLE
Loss function in DCVC-RT Model

### ISSUE THREAD

**Issue by Qiuyu-Li276** [microsoft/DCVC#108]
For the distortion calculation in YUV444, do you use "loss_yuv = mse_y * 6.0 + mse_u + mse_v", or "loss_yuv = mse_y * 4.0 + mse_u + mse_v" or "loss_yuv = mse_y + mse_u + mse_v"?

**Comment by nwwwww** [comment]
Hi, join the group and discuss it together


<img width="1031" height="1449" alt="Image" src="https://github.com/user-attachments/assets/63b78e1a-dde0-4629-afbb-70731cb9d2f8" />

**Comment by silidada** [comment]
Hello, I’ve also been working on the retraining of DCVC-RT lately and would like to discuss it with you. However, the QR code you provided has expired—could you please send a new one? @nwwwww

**Comment by nanli320** [comment]
@nwwwww Could you please share a new QR code? Thanks.

**Comment by yaohualibin** [comment]
We have released DCVC-UF code, including the training code. Please have a try.
